src/cartservice/Program.cs

```diff
@@ -41,6 +46,9 @@ class ServerOptions
 
             [Option('r', "redis", HelpText = "The ip of redis cache")]
             public string Redis { get; set; }
+
+            [Option("projectId", HelpText = "The ProjectId to which telemetry will flow")]
```

Comment:
I'd call this string

    "(optional) GCP project ID to send Stackdriver telemetry data"